recipes/prophet/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "prophet" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e682e8ea6e1ee26a92cf289f207d539f30e44879126c128ff8f4e360eb25a8b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This looks like a pure Python package. No way to do this as `noarch: python`?

Reply:
Would the `binary_has_prefix_files` bit below work with `noarch`?

Reply:
Any chance they could just be `*.pkl`?

Reply:
Well I think part of the question is, does binary prefix replacement work with `noarch` packages?

Reply:
@dopplershift thanks for the help! I tried the `noarch: python` and it didn't work because although this package is pure python, during build it uses its dependency pystan to compile a model binary that is then stored in the pickle. It seems that with `noarch: python` that build didn't happen and the package failed to install.

Reply:
Can you give it another shot? It looks like the first time you tried in 1058c2a46a389827e60f2c1b2b147113a450d2e1, you added `noarch: python`, but you also removed the `script` line, which is what tells it to run `setup.py`. Try only adding the `noarch: python`.

Reply:
Thanks, and forgive my ignorance, for some reason I thought it was supposed to replace script!

Reply:
> Would the `binary_has_prefix_files` bit below work with `noarch`?

To answer my own question, this seems to be working fine with `noarch`

Reply:
:tada: